test/wh_test_clientserver.c

```diff
@@ -199,13 +199,23 @@ int whTest_ClientServerSequential(void)
             .context = nfc,
             .config = nf_conf,
     }};
+    whNvmContext nvm[1] = {{0}};
+
+    crypto_context crypto[1] = {{
+            .devId = INVALID_DEVID,
+    }};
 
     whServerConfig                  s_conf[1] = {{
        .comm_config = cs_conf,
-       .nvm_config = n_conf,
+       .nvm = nvm,
+       .crypto = crypto,
     }};
     whServerContext                server[1] = {0};
 
+    wolfCrypt_Init();
+    wc_InitRng_ex(crypto->rng, NULL, crypto->devId);
+    wh_Nvm_Init(nvm, n_conf);
```

Comment:
check return values for these three functions
```suggestion
    WH_TEST_RETURN_ON_FAIL(wolfCrypt_Init());
    WH_TEST_RETURN_ON_FAIL(wc_InitRng_ex(crypto->rng, NULL, crypto->devId));
    WH_TEST_RETURN_ON_FAIL(wh_Nvm_Init(nvm, n_conf));
```